dockernet/start_network.sh

```diff
@@ -44,7 +44,7 @@ for chain in STRIDE ${HOST_CHAINS[@]}; do
 done
 
 # Start the chain and create the transfer channels
-bash $SRC/start_chain.sh 
+DOCKER_DEBUG=$DOCKER_DEBUG bash $SRC/start_chain.sh
```

Comment:
Do you need to pass the argument again here?

Reply:
No..